packages/office-addin-lint/src/cli.ts

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as commander from "commander";
+import { logErrorMessage } from "office-addin-cli";
+import * as commands from "./commands";
+import * as defaults from "./defaults";
+
+commander.name("office-addin-lint");
+commander.version(process.env.npm_package_version || "(version not available)");
+
+commander
+    .command("lint")
```

Comment:
I think we should make this the "check" command so that this is used via the package.json scripts as:
` "lint": "office-addin-lint check"`